modules/talkinghead.mjs

```
@@ -2890,6 +2895,16 @@ class TalkingHead {
       o.audio = r.audio;
     }
 
+    // Blend shapes animation
+    if (r.anim && r.anim.name) {
+      let animObj = this.animFactory(r.anim, false, 1, 1, true);  
+      if (!o.anim) {
+        o.anim = [ animObj ];
+      } else {
+        o.anim.push(animObj);
+      }
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)